data_dumper/Cargo.toml

```diff
@@ -8,7 +8,8 @@ anyhow = { workspace = true }
 dedicated_executor = { workspace = true }
 features = { workspace = true }
 fs-err = { workspace = true }
-log = { workspace = true }
+tracing = { workspace = true }
+logging = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order